config.gradle

```diff
@@ -27,12 +27,12 @@ project.ext.fluentui_persona_versionid = '0.1.16'
 project.ext.fluentui_progress_versionid = '0.1.9'
 project.ext.fluentui_icons_versionid = '0.1.7'
 project.ext.fluentui_notification_versionid = '0.1.12'
-project.ext.FluentUI_versionid = '0.1.35'
+project.ext.FluentUI_versionid = '0.1.36'
 project.ext.fluentui_calendar_version_code = 28
 project.ext.fluentui_controls_version_code = 19
 project.ext.fluentui_core_version_code = 50
```

Comment:
update core version as this PR //github.com/microsoft/fluentui-android/pull/513/files has changes in core module